lensless/recon.py

```diff
@@ -261,7 +262,7 @@ def reset(self):
         return
 
     @abc.abstractmethod
-    def _update(self):
+    def _update(self, iter):
```

Comment:
perhaps worth mentioning in CHANGELOG that added `iter`. 

This is only needed because of unrolled algorithms though right?

Reply:
Yes, for all the "standard" algorithm it is unused.